content/concepts/multiplex/yamux.md

```diff
@@ -1,15 +1,33 @@
 ---
-title: "yamux"
+title: "Yamux"
 description: "yamux is a multiplexing protocol designed by Hashicorp."
 weight: 180
 ---
 
-[yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed by [Hashicorp](https://www.hashicorp.com/).
-yamux offers more sophisticated flow control than mplex, and can scale to thousands of multiplexed streams over a
-single connection.
+## What is Yamux?
 
-{{< alert icon="💡" context="note">}}
-This section is incomplete, and many of the articles are stubs. To help fill in
-the gaps, please see the issues linked in each article to add your input and
-help us prioritize the outstanding work.
+[Yamux](https://github.com/hashicorp/yamux) (Yet another Multiplexer) is a multiplexing
+protocol initially designed by [Hashicorp](https://www.hashicorp.com/) for Go, built over
+TCP. Yamux provides a high-level interface inspired
+by [SPDY](https://en.wikipedia.org/wiki/SPDY), but it is not interoperable with it.
+
```

Comment:
First we should say that Yamux allows us to have multiple streams in parallel on a single TCP connection.